app/javascript/channels/project_channel.ts

```diff
@@ -0,0 +1,23 @@
+import $ from "jquery"
+import consumer from "./consumer"
+
+consumer.subscriptions.create(
+  {
+    channel: "ProjectChannel",
+    organization_id: $("#organization_subscription_identifier").val(),
```

Comment:
If jquery isn't cool and you prefer a more modern/react-friendly alternative, this _might_ be its only usage.

Reply:
Oo good call - going to make a follow-up task to remove jQuery and update this logic.